src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -27,74 +52,603 @@ const ActorInit En_Dns_InitVars = {
+        case ENDNS_PARAMS_7_0:
+            return 0x1D1;
+
+        case ENDNS_PARAMS_7_1:
+            return 0x1D2;
+
+        case ENDNS_PARAMS_7_2:
+            return 0x1D3;
+
+        case ENDNS_PARAMS_7_3:
+            return 0x1D4;
+    }
+
+    return 0;
+}
+
+s32 func_8092CAD0(EnDns* this, GlobalContext* globalCtx) {
+    s32 ret = false;
+
+    if (this->unk_2C6 & 7) {
+        if (func_800B84D0(&this->actor, globalCtx)) {
+            func_8013AED4(&this->unk_2C6, 0, 7);
+            this->unk_2C6 &= ~0x10;
+            if (ENDNS_PARAMS_4000(&this->actor)) {
+                this->unk_2F0 = 0.0f;
+                if (this->unk_2D2 != 0) {
+                    this->unk_2F0 = this->skelAnime.animCurrentFrame;
+                    func_8092C63C(this, 2);
+                }
+                this->unk_2DA = this->actor.world.rot.y;
+            }
+            this->unk_1E0 = func_8092C9BC(this, globalCtx);
+            this->actionFunc = func_8092D4D8;
+            ret = true;
+        }
+    }
+    return ret;
+}
+
+s32 func_8092CB98(EnDns* this, GlobalContext* globalCtx) {
+    s32 phi_v1 = 0;
+
+    if (globalCtx->csCtx.state != 0) {
+        if (!(this->unk_2C6 & 0x80)) {
+            this->unk_2C8 = func_8092CA74(this);
+            this->actor.flags &= ~0x1;
+            func_8013AED4(&this->unk_2C6, 0, 7);
+            this->unk_2C6 |= 0x80;
+            this->unk_1D8 = 0xFF;
+        }
+        phi_v1 = 1;
+    } else if (this->unk_2C6 & 0x80) {
+        this->actor.flags |= 1;
+        func_8013AED4(&this->unk_2C6, 3, 7);
+        this->unk_2C6 &= ~0x80;
+    }
+    return phi_v1;
+}
+
+s32 func_8092CC68(GlobalContext* globalCtx) {
+    Player* player = PLAYER;
+    s32 pad[2];
+    s32 ret = false;
+    s16 bgId;
+
+    if (!func_801690CC(globalCtx) && (player->actor.bgCheckFlags & 1) && (player->transformation != PLAYER_FORM_DEKU)) {
+        bgId = player->actor.floorBgId;
+        if (func_800C99AC(&globalCtx->colCtx, player->actor.floorPoly, bgId) != 4) {
+            ret = true;
+        }
+    }
+
+    return ret;
+}
+
+s32 func_8092CCEC(EnDns* this, GlobalContext* globalCtx) {
+    Player* player = PLAYER;
+    s32 pad;
+    Vec3f sp3C = player->actor.world.pos;
+    Vec3f sp30 = this->actor.world.pos;
+    s16 sp2E;
+
+    Math_Vec3f_Copy(&sp30, &this->actor.world.pos);
+    Math_Vec3f_Copy(&sp3C, &player->actor.world.pos);
+    this->unk_2D6 = Math_Vec3f_Yaw(&D_801D15B0, &sp3C);
+    this->unk_2D4 = Math_Vec3f_Yaw(&D_801D15B0, &sp30);
+    this->unk_2EC = Math_Vec3f_DistXZ(&sp30, &D_801D15B0);
+    sp2E = Math_Vec3f_Yaw(&D_801D15B0, &sp3C);
+    sp2E -= Math_Vec3f_Yaw(&D_801D15B0, &sp30);
+    this->unk_2D8 = (Rand_ZeroOne() * 182.0f) + 182.0f;
+    this->unk_2D8 = (sp2E > 0) ? this->unk_2D8 : -this->unk_2D8;
+    this->unk_2D0 = 0x28;
+    this->actor.shape.shadowDraw = func_800B3FC0;
+    return 1;
+}
+
+s32 func_8092CE38(EnDns* this) {
+    static s32 D_8092DE00[] = {
+        8,
+        8,
+        9,
+    };
```

Comment:
```suggestion
    static s32 D_8092DE00[] = {
        8,
        8,
        9
    };
```
and then the formatter should put it on one line.